ui/src/components/DashboardGlobalHealth.js

```diff
@@ -0,0 +1,58 @@
+import React, { useMemo } from 'react';
+import { useAlerts } from '../containers/AlertProvider';
+import styled from 'styled-components';
+import { spacing } from '@scality/core-ui/dist/style/theme';
+import DashboardAlerts from './DashboardAlerts';
+
+const GlobalHealthContainer = styled.div`
+  display: flex;
+  height: 100%;
+  padding: 0 ${spacing.sp24} 0 ${spacing.sp24};
+
+  .datacenter {
+    flex: 1 0 15%;
+  }
+  .healthbar {
+    flex: 1 0 35%;
+  }
+  .alerts {
+    flex: 1 0 20%;
+  }
+  .activity {
+    flex: 1 0 20%;
+  }
+  & > div {
+    display: flex;
+    align-items: center;
+    &:not(:first-of-type):before {
+      content: '';
+      position: relative;
+      margin: 0 ${spacing.sp32} 0 ${spacing.sp32};
+      height: ${spacing.sp32};
+      width: ${spacing.sp2};
+      background-color: ${(props) => props.theme.backgroundLevel1};
+    }
+  }
+`;
+
+const DashboardGlobalHealth = () => {
+  const alerts = useAlerts({});
+  const leafAlerts = useMemo(
+    () => alerts?.alerts.filter((alert) => !alert.labels.children) || [],
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [JSON.stringify(alerts?.alerts)],
+  );
+
+  return (
+    <GlobalHealthContainer>
+      <div className="datacenter">Datacenter</div>
```

Comment:
we should hide/remove this `datacenter` div because we are not ready to display this information :)